vpr/src/base/vpr_types.h

```diff
@@ -977,6 +977,12 @@ enum e_router_algorithm {
     TIMING_DRIVEN,
 };
 
+enum e_node_reorder_algorithm {
```

Comment:
Should put an rr_ in the name of this type -- node_reorder is a bit vague.  Should also comment what this enumerated type is for.

Reply:
Done.